src/pages/SummaryPage.js

```diff
@@ -60,6 +60,7 @@ export default function SummaryPage() {
       console.log(error);
```

Comment:
In line 44 and in line 60 you still got a console.log 
They should probably be console.error(error)

And you could refactor line 21 and line 24 into one without that variable

Reply:
Ah I was sure I changed it, ty for pointing it out.